code/cmdline/cmdline.cpp

```diff
@@ -1036,6 +1036,105 @@ char *cmdline_parm::str()
 	return args;
 }
 
+#ifdef SCP_UNIX
+// Return a vector with all filesystem names of "parent/dir" relative to parent.
+// dir must not contain a slash.
+static SCP_vector<SCP_string> unix_get_single_dir_names(SCP_string dir, SCP_string parent)
```

Comment:
Minor style remark: I would recommend switching `dir` and `parent`. The final path will look like `<parent>/<dir>` so having `parent` as the first parameter would make more sense.